ui/desktop/src/main.ts

```diff
@@ -151,6 +151,11 @@ async function ensureTempDirExists(): Promise<string> {
 
 if (started) app.quit();
 
+if (process.env.ENABLE_PLAYWRIGHT) {
+  console.log('[Main] Enabling Playwright remote debugging on port 9222');
+  app.commandLine.appendSwitch('remote-debugging-port', '9222');
```

Comment:
if it's a command-line flag, why use an environment variable? does `electron-forge start -- --remote-debugging-port=9222` not do the same?